wurst/core/models/issues.py

```diff
@@ -9,8 +9,15 @@
 from wurst.core.consts import StatusCategory
 
 
+class NounsMixin(object):
+    def get_nouns(self):
+        nouns = set(self.nouns.split())
+        nouns.add(self.slug)
```

Comment:
There's spaces in some names, is that kosher?

    wurst add "in progress" task "Try to cope with spaces in nouns"